### DISCUSSION TITLE
盆友们中午好啊

### DISCUSSION THREAD

**Discussion by cyq20091029** [share121/inter-knot#3387] [其他]
<img width="360" alt="截屏2024-08-21 13 52 14" src="https://github.com/user-attachments/assets/6e7a5e2e-0b63-49b0-8e0a-76c5d7a5112f">

**Comment by ALXY-12** [comment]
不开心<img src="https://bbs-static.miyoushe.com/static/2024/07/31/cd2ca0d9143e10e588e0a0191bf71c3b_5335236511308035335.png" alt="素裳 喷火" width="75" ><img src="https://bbs-static.miyoushe.com/static/2024/07/31/cd2ca0d9143e10e588e0a0191bf71c3b_5335236511308035335.png" alt="素裳 喷火" width="75" ><img src="https://bbs-static.miyoushe.com/static/2024/07/31/cd2ca0d9143e10e588e0a0191bf71c3b_5335236511308035335.png" alt="素裳 喷火" width="75" >

**Comment by ybbgh** [comment]
3抽白露小保底，30抽大保底花火<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Comment by opqphp** [comment]
不好
![IMG_20240821_18442058](https://github.com/user-attachments/assets/d816ffd6-b7d3-410f-8b77-8d334c45ad96)
![illust_116799150_20240819_000712](https://github.com/user-attachments/assets/9c245b13-6177-402d-9ebd-226ab810275e)
